src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -874,11 +874,10 @@ internal static bool IsKeyword(string str)
             return false;
         }
 
-        internal void SkipNewlines(bool skipSemis, bool v3)
+        internal void SkipNewlines(bool skipSemis)
         {
-        // We normally don't create any tokens here, but the V2 tokenizer api returns newline tokens,
-        // so we create them when asked to create them.
-
+            // We normally don't create any tokens here, but the V2 tokenizer api returns newline tokens,
+            // so we create them when asked to create them.
```

Comment:
It seems the comment is out of date - currently we can create Newline, Semi and LineContinuation tokens.

Reply:
Agreed, I'll remove it.

Reply:
Updated.